app/Models/ForumPost.php

```
@@ -0,0 +1,154 @@
+            : Storage::url($this->image_path);
+    }
+
+    protected static function booted(): void
+    {
+        static::creating(function (ForumPost $post) {
+            if (empty($post->slug)) {
+                $baseSlug = Str::slug($post->title) ?: 'post';
+                $slug = $baseSlug;
+                $counter = 1;
+
+                while (static::where('slug', $slug)->exists()) {
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\ForumPost::where().


(staticMethod.notFound)